src/pages/TeamPage/TeamPage.tsx

```diff
@@ -198,7 +201,43 @@ export const TeamPage: React.FC = () => {
 
   // Option 1 - Remove user from Team
 
+  const canRemoveFromTeam = (member: FetchTeamMembersResponse) => {
+    const currentUser = user;
```

Comment:
Why is this variable needed?

Reply:
In my honest opinion, it makes it clearer that the user we're referring to IS the current user, and not the "member" one.

I do get where you're getting to - user might've sufficed, but I thought this would add more clarity.